src/server/db_slice.cc

```diff
@@ -1338,4 +1338,19 @@ void DbSlice::ResetUpdateEvents() {
   events_.update = 0;
 }
 
+void DbSlice::TrackKeys(const facade::Connection::WeakRef& conn, const ArgSlice& keys) {
+  if (conn.IsExpired()) {
```

Comment:
This is impossible, I'd do `DCHECK(!conn.Expired())` (in the next pr)

Reply:
> This is impossible, I'd do `DCHECK(!conn.Expired())` (in the next pr)

got it, will do it there.